pre_commit_hooks/mixed_line_ending.py

```diff
@@ -0,0 +1,250 @@
+import argparse
+import logging
+import os
+import re
+import sys
+
+from enum import Enum
```

Comment:
Well I think I miss something. The tests [pass with Python 2.7](https://travis-ci.org/nagromc/pre-commit-hooks/jobs/254972658). But I haven't declared anything relative to `enum`. Yet, it [seems to be "imported"](https://travis-ci.org/nagromc/pre-commit-hooks/jobs/254972658#L312) with some magic.

Is it some kind of pre-built environment provided by `tox`?